cli/cli.ts

```diff
@@ -4674,22 +4687,32 @@ function buildJResSpritesCoreAsync(parsed: commandParser.ParsedCommand) {
 
                 let storeIcon = false
```

Comment:
Storing the icon is better for perf; for context the icon is the PNG format of the image vs the data which is in our image format.

We don't do it for arcade because the palette can change.